proactive-support-bot.py

```diff
@@ -56,6 +57,13 @@ def handle_fileuploadtest(client, event):
         channels=channel,
         content="Hello, World")
 
+def handle_cluster(client, event):
+    channel = event['event']['channel']
+    cluster = event['event'].get('text').split()[-1]
+    account = ocm.cluster_to_account(os.environ['PYOCM_SECRETpyocm'], cluster)
+    return client.chat_postMessage(
+            channel=channel,
+            text='Based on {0}, we found SFDC Account {1}'.format(cluster, account))
```

Comment:
What is SFDC?  Can we spell it out?

Reply:
Salesforce, I expanded it to Red Hat Customer Account as that is technically more accurate.